io.github.yuxshao.ptcollab.metainfo.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.yuxshao.ptcollab</id>
+  
+  <name>pxtone Collab</name>
+  <summary>Sample-based music editor where you can collaborate with friends!</summary>
+  
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
+
+  <screenshots>
+    <screenshot type="default">https://cdn.discordapp.com/attachments/1033076370420535398/1212564674097447032/Captura_de_pantalla_de_2024-02-28_21-51-40.png?ex=65f24bda&is=65dfd6da&hm=bc8aeeebc65e67e3c64b7d5a8fee59d103810b90ed6a2b561216fc87ede10233&</screenshot>
```

Comment:
url have to be inside `<image>`

Don't use discord as your storage. It's unreliable at best.

Reply:
Oh ok! what service do I use then? Can I use protondrive?

Reply:
this git repo? or another one?

Reply:
images are in github now